lua/kubectl/views/filter/init.lua

```diff
@@ -31,7 +34,77 @@ local function save_history(input)
   state.filter_history = result
 end
 
+function M.filter_label()
+  local state = require("kubectl.state")
+  local instance = vim.deepcopy(state.instance)
```

Comment:
Do we need to deepcopy here? this object can be really big!

Reply:
it's working without deepcopy. I removed it and added highlights